tests/conftest.py

```diff
@@ -24,4 +26,18 @@ def synapse_manifest():
 
 @pytest.fixture()
 def local_manifest():
-    return pd.read_csv("tests/data/mock_manifests/local_manifest.csv")
\ No newline at end of file
+    return pd.read_csv("tests/data/mock_manifests/local_manifest.csv")
```

Comment:
I suggest you use the new `get_data_frame()` helper function in `helpers` fixture that was just merged.

Reply:
You might have to merge `develop` into your branch first.